docs/concepts/extension_points.rst

```diff
@@ -1,16 +1,17 @@
 
-Open edX platform development follows the `Open-Closed Principle`_: we want Open edX to be an extensible platform that allows developers to build extensions that integrate with the core of the platform. This allows the core to remain small, while volatile extensions remain in the periphery.
+Open edX platform development aims to follow the `Open-Closed Principle`_: developers should be able to extend the platform to suit their needs ("open") without relying upon the details of the platform's internal implementation ("closed"). This makes it easier to maintain and refactor the platform's core, while enabling experimentation and volitility in its extensions.
 
-As you can see in this document, there are many different ways to integrate with Open edX. However, we know that there are still some features/integrations that are not possible today without modifying the core. If you have such a need, please consider proposing a new extension point in the core that would make possible the functionality you have in mind. When you submit a pull request for a new extension point, be sure to include a change to this file to document your new extension point. (Doing so will also notify reviewers that want to help with making the platform more extensible.)
+There are many different ways to integrate with Open edX. If you would like to build a feature or integration that does **not** seem to be supported by these integration methods, consider proposing a new extension point in the core that would make possible the functionality you have in mind. When you submit a pull request for a new extension point, be sure to include a change to this file to document your new extension point. (Doing so will also notify reviewers that want to help with making the platform more extensible.)
 
-Throughout this document, we will refer to the **Status** (**Adoption** and **Completion**) of each specific integration point. The Completion refers to how complete and stable an integration point is: either "Limited" (incomplete, or unstable) or "Stable" (complete and stable enough for general use in some or all cases). Adoption shows how the integration point is currently being used, and whether or not it should be used in the future:
+Throughout this document, we will refer to the **Status** of each specific extension point:
+
+* ✅ **Stable**: We actively recommend that developers use these. Core developers strive to avoid breaking these type extensions. When breaking changes are necessary, they will go through the `standard deprecation process <https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html>`_.
+* ⚠️ **Provisional**: We cautiously recommend these for use when Stable extension points are not sufficient. They may or may not be as well-documented. Core deevlopers may need to introduce breaking changes from time to time, but will ensure that these changes are communicated via the deprecation process, the named release notes, or both.
```

Comment:
```suggestion
* ⚠️ **Provisional**: We cautiously recommend these for use when Stable extension points are not sufficient. They may or may not be as well-documented. Core developers may need to introduce breaking changes from time to time but will ensure that these changes are communicated via the deprecation process, the named release notes, or both.
```

Reply:
I just want to clarify that despite being "provisional" these extension points will still go through the DEPR process when being changed?

Reply:
Yes, breaking changes to provisional APIs should still go through the DEPR process.